webrender/src/frame.rs

```diff
@@ -190,6 +154,57 @@ impl StackingContextHelpers for StackingContext {
     }
 }
 
+pub struct DisplayListTraversal<'a> {
+    pub display_list: &'a Vec<DisplayItem>,
```

Comment:
As a uber-nit: this can probably be `&'a [DisplayItem]` to save an extra level of indirection.